src/routes/event-match.tsx

```diff
@@ -90,22 +102,54 @@ const matchWithVideoStyle = css`
   }
 `
 
+const leftColumnStyle = css`
+  grid-area: leftColumn;
+  display: grid;
+  grid-gap: 1.5rem;
+  justify-self: center;
+  @media (max-width: 540px) {
+    justify-self: stretch;
+  }
+`
+
+const offlineDisplayInfo = css`
+  grid-area: topArea;
+  display: grid;
+  grid-template-areas:
+    'iconArea title'
+    'detail detail';
+  justify-self: center;
```

Comment:
Can you add `align-items: center` so that the "No connection" text is vertically centered with the icon?